pandas/io/sql.py

```diff
@@ -1588,9 +1594,13 @@ def _create_table_setup(self):
             create_tbl_stmts.append(
                 f"CONSTRAINT {self.name}_pk PRIMARY KEY ({cnames_br})"
             )
-
+        if self.schema:
+            schema_to_add = self.schema + "."
```

Comment:
Granted I'm no SQL expert, but perhaps `schema_name` would be a better variable name?